.github/workflows/tests.yml

```
@@ -240,90 +273,90 @@ jobs:
       - name: Load and Start Appwrite
         run: |
-          sed -i 's/traefik/localhost/g' .env 
-          docker load --input /tmp/${{ env.IMAGE }}.tar          
+          docker load --input /tmp/${{ env.IMAGE }}.tar
+          sed -i 's/_APP_OPTIONS_ABUSE=disabled/_APP_OPTIONS_ABUSE=enabled/' .env
           docker compose up -d
-          sleep 10
-      - name: Install Oha
+          sleep 30
+
+      - name: Run Projects tests with dev keys in ${{ matrix.tables-mode }} table mode
```

actionlint (1.7.7)

property "tables-mode" is not defined in object type {}

(expression)